index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Odin Recipes</h1>
+    <ul>
+        <li><a href="white.html">white-rice</a> </li>
+        <li><a href="rosemary.html">rosemary</a></li>
+        <li><a href="sufganiyot.html">sufganiyot</a></li>
+    </ul>
+    <h1>
+        Rosemary Potatoes with Caramelized Onions
+        <img src="../images/rosemary.jpg">
+    </h1>
+    <h2>How to prepare</h2>
+    <p>This is one of meals that that needs time to prepare as it contains more steps to be followed.</p>
+<h3>Ingredients</h3>
+<ul>
+    <il>2 pounds small red potatoes quartered
+    </il>
+<il>2 garlic cloves minced</il>
+<il>1 tablespoon olive oi</il>
+<il>2 teaspoons minced fresh rosemary or 1/2</il>
+<il>teaspoon dried rosemary crushed</il>
+<il>1/2 teaspoon minced fresh thyme or 1/8</il>
+<il>teaspoon dried thyme</il>
+<il>1/4 teaspoon salt</il>
+<il>1/4 teaspoon pepper</il>
+<il>2 large sweet onions chopped</il>
+<il>2 tablespoons olive oil</il>
+<il>1 tablespoon sugar</il>
+<il>2 teaspoons balsamic vinegar</il>
+</ul>
+<h4>Steps to follow</h4>
+<ol>
+<li>In a large bowl, combine the first 7 ingredients; toss to coat. Transfer to a greased 15x10x1-in. baking pan.</li>
+<li>Bake at 425° for 45-50 minutes or until potatoes are tender, stirring once.</li>
+<li>Meanwhile, in a large skillet, saute onions in oil until softened. Stir in sugar. Reduce heat to medium-low; cook for 30-40 minutes or until deep golden brown, stirring occasionally. Stir in vinegar. </li>
+<li>     Transfer roasted potatoes to a large bowl; stir in caramelized onions.
+</li>
+</ol>
```

Comment:
It's better to wrap the Html elements in semantic tags like divs, sections, etc.

Reply:
@kidde60 it would be better to explain why you require changes, maybe it's obvious for you but not for everyone